dev-tools/ls-apis/src/api_metadata.rs

```diff
@@ -186,6 +200,24 @@ impl TryFrom<RawApiMetadata> for AllApiMetadata {
     }
 }
 
+/// Describes how an API in the system manages drift between client and server
+#[derive(Default, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "snake_case")]
+#[serde(tag = "versioned_how", content = "versioned_how_reason")]
+pub enum VersionedHow {
+    /// We have not yet determined how this API will be versioned.
+    #[default]
```

Comment:
Is this `#[default]` required?

Reply:
I don't think so.  I've removed it.  (I think it was from when I was initially filling in the field.)